Vagrantfile

```diff
@@ -0,0 +1,157 @@
+      # Current issue with rvm: https://github.com/rvm/rvm/issues/4068
+      # use this patch until fixed
+      curl -sSL https://raw.githubusercontent.com/wayneeseguin/rvm/stable/binscripts/rvm-installer | bash -s stable --ruby=2.4
+      # uncomment and use below after fixed
+      # curl -sSL https://get.rvm.io | bash -s stable --ruby=2.4
+      source /home/ubuntu/.rvm/scripts/rvm
+      echo "${GREEN}COMPLETED INSTALLING RVM & RUBY${NC}"
+      echo ' '
+      echo "${PURPLE}INSTALLING GEMS${NC}"
+      gem install bundler --no-rdoc --no-ri
+      # also install libcurl3, libcurl3-gnutls, & libcurl4-openssl-dev
+      # which are needed by the "curb" gem, used in some 3rd party rake tasks--tho not actually sure about neo4j
+      yes | sudo apt-get install libcurl3 libcurl4-openssl-dev
+      bundle install
+      gem install overcommit --no-rdoc --no-ri
+      overcommit --install
+      echo "${GREEN}COMPLETED INSTALLING GEMS${NC}"
+  SHELL
+
+  unless Dir["./db/neo4j/development"].any?
+    # With default configuration, Neo4j only accepts local connections (which, in this case, means local to the VM).
+    # Vagrant has been set up to forward connections local to the VM to the appropriate ports on the host machine.
+    # As such, we need to change neo4j's conf to accept non-local connections if we want to be able to connect to
+    # neo4j from a browser on the host machine:
+    update_neo4j_conf = <<-TEXT
+      require 'fileutils'
+      require 'tempfile'
+
+      t_file = Tempfile.new('neo4j.conf')
+      File.open('./db/neo4j/development/conf/neo4j.conf', 'r+').each do |line|
+        case line.chomp
+        when '#dbms.connectors.default_listen_address=0.0.0.0'
+          t_file.puts 'dbms.connectors.default_listen_address=0.0.0.0'
+        when 'dbms.connector.bolt.listen_address=localhost:7472'
+          t_file.puts 'dbms.connector.bolt.listen_address=:7472'
+        when 'dbms.connector.http.listen_address=localhost:7474'
+          t_file.puts 'dbms.connector.http.listen_address=:7474'
+        when 'dbms.connector.https.listen_address=localhost:7473'
+          t_file.puts 'dbms.connector.https.listen_address=:7473'
+        else
+          t_file.puts line
+        end
+      end
+      t_file.close
+      FileUtils.mv(t_file.path, './db/neo4j/development/conf/neo4j.conf')
+    TEXT
+
+    config.vm.provision "install neo4j", type: "shell", privileged: false,
+      keep_color: true, inline: <<-SHELL
+        PURPLE='\033[1;35m'; GREEN='\033[1;32m'; NC='\033[0;0m'
+        echo "${PURPLE}INSTALLING NEO4J INTO 'db/neo4j/development'${NC}"
+        cd ~/#{project_name}
+        rake neo4j:install[community-latest]
+        ruby -e "#{update_neo4j_conf}"
+        echo "${GREEN}COMPLETED INSTALLING NEO4J INTO 'db/neo4j/development'${NC}"
+    SHELL
+  end
+
+  config.vm.provision "bootstrapping complete", type: "shell", privileged: false,
+    keep_color: true, run: "always", inline: <<-SHELL
```

Comment:
Finally, output a VM start up message. The `run: "always"` option tells vagrant to run these shell scripts each time `vagrant up` is used to start the VM. If `run: "always"` isn't included, vagrant will only run the commands the first time you `vagrant up` as part of the initial provisioning process.